src/pages/auth/Signup.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { Link } from 'react-router-dom';
+
+function Signup() {
+  return <>
+   <div className="flex-center height-vh-100">
+        <form className="form-auth flex-center flex-col bx-shadow">
+            <div className="form-logo wd-100">
+                <Link to="/"> 
+                    <div className="nav-logo flex-center flex-col">
+                        <i className="fas fa-jedi nav-logo-icon"></i>
+                        <span className="nav-logo-txt">Jedi Store</span>
+                    </div>
+                </Link>
+            </div>
```

Comment:
it's jedi note it or jedi store?

Reply:
;-) forgot to rename as its taken from ecom , thank you for pointing out .